src/validation.cpp

```diff
@@ -1379,6 +1570,7 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptPackage(const Package& package,
                 // future.  Continue individually validating the rest of the transactions, because
                 // some of them may still be valid.
                 quit_early = true;
+                package_state.Invalid(PackageValidationResult::PCKG_TX, "transaction failed");
```

Comment:
no results are being pushed here, which seems to mean we'll hit `NONFATAL_UNREACHABLE` in the switch statement in `submitpackage` RPC with a tx that doesn't hit the above two conditions.

Reply:
seems to fix all my ephemeral anchor tests if I add this, since I'm hitting this path quite a bit